MastodonSDK/Sources/MastodonUI/View/Content/StatusView+ViewModel.swift

```diff
@@ -315,7 +315,6 @@ extension StatusView.ViewModel {
                 statusView.contentMetaText.configure(
                     content: content
                 )
-                statusView.contentMetaText.textView.accessibilityLabel = content.string
```

Comment:
What's the reason for this? The `accessibilityCustomActions` aren't a replacement for `accessibilityLabel` from my understanding. Or am I wrong?

Reply:
Ah, sorry, nevermind, that's what you meant with

> Removes the duplicate accessibility label which caused post text to be read twice

right?

Reply:
That’s correct.